src/pages/Index.js

```diff
@@ -17,38 +19,100 @@ const modalDescriptionInput = document.querySelector(
   "#modal-description-input"
 );
 
+// AVATAR
+const profileAvatarContainer = document.querySelector(
+  ".profile__image-overlay"
+);
+const editProfileSaveButton = document.querySelector(
+  "#edit-profile-modal-save-button"
+);
+
 // NEW CARD MODAL
 const cardTitleInput = document.querySelector("#card-title-input");
 const cardUrlInput = document.querySelector("#card-url-input");
 const addCardModal = document.querySelector("#add-card-modal");
 const addCardForm = document.forms["modal-add-form"];
 const addCardButton = document.querySelector("#profile-add-button");
-
 const cardSelector = "#card-template";
 
+// FORMS
+const forms = document.querySelectorAll(settings.formSelector);
+
+/*                                       */
+/*                Api.JS                 */
+/*                                       */
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "5f4ef148-bee1-40be-bb22-88d989208c73",
+    "Content-Type": "application/json",
+  },
+});
+
+/*                                       */
+/*                Cards                  */
+/*                                       */
+
+function createCard(cardData) {
+  const addCard = new Card(
+    cardData,
+    cardSelector,
+    handleImageClick,
+    handleLikeClick,
+    handleDeleteClick,
+  );
+  return addCard.getView();
+}
+
+api
+  .getInitialCards()
+  .then((cards) => {
+    console.log(cards);
+    cards.forEach((card) => {
```

Comment:
Needs correction: Here you should use `renderItems` method of the section class. For that you can move `items` parameter to it instead of the constructor `cardSection.renderItems(cards);`